### PR TITLE
Adding Romanian language

### PR DESCRIPTION
**Pull request by vladutilie**
Hi,

I've translated the website in Romanian language, following the [How to translate](https://github.com/wp-cli/wp-cli.github.com#how-to-translate) section.

Best regards,
Vlad

### CODE REVIEW

**Comment by schlessera** [comment]
Thanks for the awesome work, @vladutilie!

One thing: You shouldn't translate the example code or app output. Translating the output of WP-CLI gives users the impression that there is a translated version of WP-CLI. Translating the source code will actually break things. For example, you've translated `## OPTIONS` and `## EXAMPLES`. These are being parsed by WP-CLI to generate the synopsis and the live help. If people use such a translated docblock, this will not work as expected.

**Comment by vladutilie** [comment]
Hi @schlessera,

Thank you for your feedback. I've fixed the translation, now should be fine.

Best regards,
Vlad

File: _config.yml [hunk 1]
```diff
@@ -58,3 +58,6 @@ languages:
 - label: Español
   iso-alpha2: es
   path: /es/
+- label: română
```
Comment (schlessera) [inline _config.yml]:
```suggestion
- label: Română
```

**Comment by schlessera** [comment]
Thanks for the PR, @vladutilie !

**Pull request merged**